packages/distribution/addon/components/cd-inquiry-dialog/inquiry-part.js

```diff
@@ -68,6 +73,22 @@ export default class CdInquiryDialogInquiryPartComponent extends Component {
           .map((edge) => ({
             question: edge.node.question.label,
             value: edge.node.value,
+            selectedOption: edge.node?.selectedOption,
+            selectedOptions: edge.node?.selectedOptions,
+          }))
+          // For single choice the value is simply the selectedOption label
+          .map((answer) => ({
+            ...answer,
+            value: answer.selectedOption
+              ? answer.selectedOption.label
+              : answer.value,
+          }))
+          // For multiple choice the value is a list of selectedOption labels
+          .map((answer) => ({
+            ...answer,
+            value: answer.selectedOptions
+              ? answer.selectedOptions.edges.map((edge) => edge.node.label)
+              : answer.value,
```

Comment:
I think we should put this into a function and then use it in the first `.map()`.